department-of-veterans-affairs/vets-website
# Update files and list titles for consistency

## Summary
Update list item titles for consistency on the the Pension Benefits form (21P-527EZ)

## Issue
[Issue in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/82855)

## How to run in local environment
1. Check out this branch locally
2. In your local environment, set the `pension_form_enabled` flipper to 'enabled' at http://localhost:3000/flipper/features/pension_form_enabled
3. Run `vets-website` and `vets-api`

## How to verify
1. Go to `http://localhost:3001/pension/application/527EZ` in your browser
2. Verify the 'List of other service names' title in the form and on the 'Review and submit' page
3. Verify the 'List of current employment' title in the form and on the 'Review and submit' page
4. Verify the 'List of previous employment' title in the form and on the 'Review and submit' page
5. Verify the 'List of unreimbursed care expenses' title in the form and on the 'Review and submit' page
6. Verify the 'List of medical expenses and other unreimbursed expenses' title in the form and on the 'Review and submit' page
7. Continue to the 'Review and submit' chapter
13. Verify that the data is viewable and editable
14. Verify that the application is able to be submitted

## Testing done

## Screenshots
### List of other service names
| Form | Review and submit |
| ----- | ----- |
|![localhost_3001_pension_apply-for-veteran-pension-form-21p-527ez_form-saved (9)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/1451d902-4ceb-4785-a764-7ca28f14021a)|![Screenshot 2024-05-23 at 10 27 10 AM](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/f5aca1f3-dad7-4fba-b570-35979b958bd6)

### List of current employment
| Form | Review and submit |
| ----- | ----- |
|![localhost_3001_pension_apply-for-veteran-pension-form-21p-527ez_form-saved (7)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/bc2aed36-49d6-439a-9bd8-5ee861eabaff)|![Screenshot 2024-05-23 at 10 30 23 AM](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/771bce06-ea6b-4d4d-9242-ce5ba1dce9fa)

### List of previous employment
| Form | Review and submit |
| ----- | ----- |
|![localhost_3001_pension_apply-for-veteran-pension-form-21p-527ez_form-saved (5)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/88c16a92-598a-45c9-b129-2f3b6379f748)|![Screenshot 2024-05-23 at 10 43 59 AM](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/01b74e66-305c-4ed6-9704-da91000bd1d4)

### List of unreimbursed care expenses
| Form | Review and submit |
| ----- | ----- |
|![localhost_3001_pension_apply-for-veteran-pension-form-21p-527ez_form-saved (8)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/ee32c94d-cf36-43fe-ba7b-a81b213279bb)|![Screenshot 2024-05-23 at 10 46 10 AM](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/1c667d93-6d94-42f9-a873-ee4324f32e14)

### List of medical expenses and other unreimbursed expenses
| Form | Review and submit |
| ----- | ----- |
|![localhost_3001_pension_apply-for-veteran-pension-form-21p-527ez_form-saved (6)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/ad60658d-2cb3-42c0-a616-d4e02daf0285)|![Screenshot 2024-05-23 at 10 47 20 AM](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/98936a76-a561-40e9-9a27-6b3c40f355ec)


## What areas of the site does it impact?
Pension Benefits Application

### Quality Assurance & Testing

### Error Handling

### Authentication
